Dev branch

awsmrud opened the pull request:
*Issue #, if available:*

*Description of changes:*

Added new use cases

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

ajaykarpur commented:
Can you please update this pull request with a description of your changes or additions?

awsmrud commented:
Made the following changes:

Added additional use cases in step 3.3 in file 'creative-writing-using-gpt-2-text-generation.ipynb'

Added a link to AWS Marketplace video related to this project in introduction section of the ipynb file.
